feature/system/gnmi/metadata/tests/lagre_set_consistency_test/README.md

```diff
@@ -0,0 +1,39 @@
+# gNMI-1.14 OpenConfig metadata consistency during large config push
+
+## Summary
+This test verify if OpenConfig metadata leaf at root is updated according to pushed config and not reverted or otherway modified if any rapid, concurrent non-write requests are served while setRequest is in process and SetResponce is not recived yet.
+
+## Topology
+dut.testbed
+
+## Procedure
+
+* prepare 2 large OpenConfig configurations that only differs in root-level netadata/annotation - 1st_LARGE_CONFIGURATION and 2nd_LARGE_CONFIGURATION. Configuration should exercise following branches of Openconfig:
+  *  /interfaces/
+  *  /network-instances/
+  *  /qos/
+  *  /system/
+  *  /routing-policy/
+  *  /components/
+  *  /sampling/
+  *  /lldp/
+  *  /lacp/
+  *  /protobuf-metadata
+* Push 1st_LARGE_CONFIGURATION and wait for SetResponce that confirm push was sucesfull.
+* DubbscribeRequest once for configuration root-level metadata and verify that it is identical corresponding root-level metadata of 1st_LARGE_CONFIGURATION.
+* For REQUEST of type (GetRequest, SubscribeRequest, CapabilityRequest)
+  * Repeat sending rapidly subsequent REQUEST requests for configuration root-level metadata (GetRequest, SubscribeRequest).
+  * While above REQUESTS are send, push SetRequest replace with 2nd_LARGE_CONFIGURATION.
+  * After SetResponse is recived confirming sucesfull push of 2nd_LARGE_CONFIGURATION:
+    * stop sending REQUEST requests
+    * wait 5 seconds
+    * SubscribeRequest once for configuration root-level metadata and verify that it is identical corresponding root-level metadata of 2nd_LARGE_CONFIGURATION.
+
+## Configuration path coverage
+* /config/protobuf-metadata
```

Comment:
I think the syntax for specifying should be `/@/openconfig-metadata:protobuf-metadata` to indicate that the root path has an 'annotation'.
ref: https://github.com/openconfig/featureprofiles/blob/5c2be080db121b560178bb214a41983f75e77480/feature/system/gnmi/metadata/tests/annotation_test/README.md

Reply:
done